### ISSUE TITLE
I wanted to suggest a new logo...

### ISSUE THREAD

**Issue by SecularSteve** [cheat-engine/cheat-engine#1958]
It's your choice if you think it's better...

![Cheat Engine H](https://user-images.githubusercontent.com/33793273/146689929-42d1f976-9ab3-45f5-b407-672f3d3e8b17.png)
![Cheat Engine no_text](https://user-images.githubusercontent.com/33793273/146689943-58fec2ac-230b-4427-844f-16c5d118a548.png)
![Cheat Engine](https://user-images.githubusercontent.com/33793273/146689947-ac99343b-ba36-4a90-aab1-dcec21aea77f.png)

Released under the CC BY-SA 4.0 ; Made in Inkscape... 

ZIP contains SVGs and PNGs...

[[SecularSteve] Cheat Engine New Logo.zip](https://github.com/cheat-engine/cheat-engine/files/7741554/SecularSteve.Cheat.Engine.New.Logo.zip)

**Comment by dsasmblr** [comment]
Some quick thoughts:

1. The shadow looks odd -- like you manually tried to create one instead of
using a global light source.
2. I don't think Segoe UI as a font fits with the icon, nor with the
category of what CE is as an application. Additionally, Segoe UI isn't a
free font, so that might pose an issue if used in any monetized context.
3. I feel there are too many notches in the gear for how small the icon is
in the application.
4. The center line just looks "meh".

Kudos for the time and effort you personally put in, but I feel if DB is
open to the icon being changed, then the graphic designers among us all
should maybe take a crack at it. Might be a fun thing for the community to
work on, vote on, etc.

**Comment by SecularSteve** [comment]
1. and 4.
I wanted to go for a "floating logo" effect, to make it look like the logo's levitating off the ground, with the added feel that the top part of it is higher than the bottom one (hence why the shadow of the centre line goes through). I tried making the shadow more "fuzzy", but I didn't really like the effect...

2.
I just picked the first font that looked good. If there are good free alternatives out there, you're free to use them instead.

3.
I have thought about deleting every second one, to make it look more like a notch... You do make a valid point here.

**Comment by Symbai** [comment]
> I wanted to suggest a new logo

What's wrong with the current?

**Comment by Hectoronian** [comment]
I would agree with @SecularSteve to open a logo and website redesign suggestion. @Symbai I don’t believe there’s an issue with the current logo itself, but it does have some outdated design elements, such as embossing. Creating a timeless logo requires careful consideration and time. Typically, I avoid redesigning a logo unless it starts to feel outdated while the associated software or service continues to advance. Also a more simpler logo is easier to animate, this is great for splash intros for tutorials.

I frequently work with clients who have a sentimental attachment to their original logos. While many of these organizations are non-profits, it’s important to help them understand that attracting more users, even in open-source projects, requires attention to every detail this includes UX/UI design, as success depends on more than just the quality of the app itself (Cheat Engine).

![logo cheat engine](https://github.com/user-attachments/assets/7717f37a-0c9d-47b0-a0de-3af84f24b0b1)

**Comment by fade2gray** [comment]
While @Hectoronian's is one of the better designs I've seen, it doesn't quite give me that that classic combined 'CE' visual effect. If it was more proportionally like the one on the [CE wiki](https://en.wikipedia.org/wiki/File:Cheat_Engine_2023_Vector_Logo.svg), it would get my vote.

**Comment by Hectoronian** [comment]
Thank you @fade2gray!! yesss I agree, I think I can try to assimilate another one with that style!
